test/fixtures/linter/rules/Render/2ControlRenderer.js

```diff
@@ -0,0 +1,11 @@
+// This file would be skipped from direct testing.
```

Comment:
Could we add a test where only the renderer is linted? I guess there wouldn't be any findings, meaning that one has to always lint the Control together with the renderer to see all findings in the renderer?

Reply:
Another scenario would be where the renderer is ignored. I wonder whether there would still be a message, as it is created through linting of a not ignored file (control).

Reply:
Thanks! I would need to update the message. My initial intention was to include only the Controls, and skip the Renders being checked in tests, but this seemed to be a wrong concept.

Now, all the files are checked. The special case is that Renders are checked twice- once "normally" as we do with the rest of the file and ones through the Controls' renderer context.

Maybe, I need to add some general deprecation to prove this, like a global variable reference

Reply:
I guess this still leaves us with the problem that you get different results for "MyControlRenderer.js" when including or excluding the associated control.

e.g. `ui5lint webapp/controls/MyControl.js webapp/controls/MyControlRenderer.js` vs. `ui5lint webapp/controls/MyControlRenderer.js`. The first command might reveal more issues with the renderer.

But maybe that's good enough for now? CC: @flovogt

Reply:
Yep, that's true!
And the main reason behind this is that without the binding between the Control and the Renderer, which is defined in the Control, we cannot guarantee that this file (`webapp/controls/MyControlRenderer.js`) is indeed a renderer.
I thinks we already discussed that we are not able to handle this scenario or if we can, it might be too expensive as we need to find the control that this renderer belongs to.

Of course, if there are some general issues in the code itself, like other deprecations, for example, they will be detected.

Reply:
LGTM. If you restrict the scan to a set of specific files you have to know this